pages/index.tsx

```diff
@@ -48,7 +48,7 @@ export default function Index(props: Props): JSX.Element {
               The open source event solution for virtual and in-person events.
             </Trans>
           </Typography>
-          <Button variant="contained">Create Event</Button>
+          <Button variant="contained">CREATE EVENT</Button>
```

Comment:
Why? It's already capitalized on the site.

Pls don't make unnecessary changes.

Reply:
By default, MUI button has the default property of `textTransform: "uppercase"` for the Button component
Now I figured out that, almost 90+ % of buttons in the open-event-frontend isn't using uppercase

To prevent using `"textTransform: "none"` again and again for this majority of cases, I configured the theme as `textTransform: 'none'` for Button  [here](https://github.com/fossasia/open-event-next/blob/cded71650c085a2fe39a16972f70378f1aa04d04/src/theme.ts#L22)

That's why I changed this text